qgepplugin/interlis/NOTES_INTERLIS_QGEP.md

```diff
@@ -0,0 +1,19 @@
+# Interlis / QGEP oddities
+
+## QGEP
+
+- `qgep_od.file.fk_data_media` doesn't have a FK constraint to `qgep_od.data_media.obj_id`
+
+## SIA405_Abwasser
+
+- `Deckel.Fabrikat` looks like a typo, shouldn't it be `Deckel.Fabrikant` ? (comment is `Name der Herstellerfirma`)
+
+## QGEP-SIA405 differences
+
+- `qgep_od.maintenance_event` has a N-M relation to `qgep_od.wastewater_structure`, making fields like `inspection.inspected_length` or `damage.distance` ambigous. Corresponding SIA405_Abwasser classes have a N-1 relation instead, which looks like a better fit.
+
+- In SIA405, some common fields from the concrete `normschachtschaden` and `kanalschaden` could be moved to the abstract `schaden` class (e.g. `distanz`, `quantifizierung1`, `quantifizierung2`...). This would match QGEP's implementation where it's like this already.
+
+- `qgep_od.maintenance_event.active_zone` has no equivalent in the interlis model.
+
+- `reach.elevation_determination` has no equivalent in the interlis model
```

Comment:
Corresponds to Haltung.Hoehenbestimmung

Reply:
I don't see this field in SIA405_Abwasser_2015_2_d-20180417.ili ?

Reply:
It's in the 3D extension

Reply:
`CLASS Haltung EXTENDS SIA405_ABWASSER_2015.SIA405_Abwasser.Haltung =  
    Hoehenbestimmung: (          !! Definiert die Hoehenbestimmung einer Haltung.
      genau,  !! Mittlerer Fehler: Sigma <= +/- 6 cm, Toleranz 3*Sigma <= +/- 18 cm
      unbekannt,
      ungenau  !! Siehe ungenau
    );
    Verlauf3D: Base.Polyline3D;  !! Anfangs-, Knick- und Endpunkte der Leitung (3D Koordinaten);
END Haltung;`